EveripediaNetwork/ep-ui
# Event admin

# Event Admin

This PR works on creating an admin page for event wikis

## How should this be tested?

1. Go to admin/dashboard/events page
2. Try out the filters, sort, and search feature
3. Promote an event then check the trending section on event page
4. Archive/Unarchive an event
5. Test the pagination to see that it works

## Linked issues

 closes https://github.com/EveripediaNetwork/issues/issues/3073
